tasks/tests/test_get_task_comments.py

```diff
@@ -34,20 +34,22 @@ def task(self, user_and_manager):
     def comments(self, user_and_manager, task):
         user, manager = user_and_manager
         comments = {}
-        for i in range(1, NUM_COMMENTS + 1):
-            comments[f'comment{i}'] = Comment.objects.create(appUser=user,
+        for i in range(1, 2 * NUM_COMMENTS_PER_USER + 1):
+            comment_user = user if i <= NUM_COMMENTS_PER_USER else manager
+            comments[f'comment{i}'] = Comment.objects.create(appUser=comment_user,
                                                              task=task,
                                                              title=f'Comment{i}',
                                                              description='This is a test comment')
-        assert len(comments) == NUM_COMMENTS
+        assert len(comments) == 2 * NUM_COMMENTS_PER_USER
         return task
 
     def test_get_relevant_comments(self, task, comments):
         fetched_comments = task.get_comments()
-        assert len(fetched_comments) == NUM_COMMENTS
+        assert len(fetched_comments) == 2 * NUM_COMMENTS_PER_USER
         assert isinstance(fetched_comments, QuerySet)
         assert all(isinstance(comment, Comment) for comment in fetched_comments)
-        assert all(comment.appUser == task.assignee for comment in fetched_comments)
+        assert all(comment.appUser == task.assignee or comment.appUser == task.created_by
+                   for comment in fetched_comments)
```

Comment:
I think that with this assert even if you will have not expected user it will pass because you have the "or" operator

Reply:
I'm not sure I understand what do you mean here

Reply:
I changed it to make more sense, let me know if that resolves what you meant.